### PR TITLE
Add Couchbase Containers FAQ

### PR DESCRIPTION
**Pull request by amarantha-k**

### CODE REVIEW

**Comment by amarantha-k** [comment]
Note that once this PR is reviewed and approved, the changes will be back ported to 6.6.

**Review by anilkumar29 — APPROVED** [review]
Reviewed!

**Comment by gorgonizer** [comment]
Should the statement:

{quote}
However, if you choose to deploy Couchbase Server on the Kubernetes platform on your own, note that Couchbase will only support product issues with container images.  
{quote}

Also mention that we would suppor tthe Couchbase Server software as well as the images?

File: modules/cloud/pages/couchbase-containers-faq.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
+= Frequently Asked Questions about Couchbase Containers
+:description: Frequently asked questions about Couchbase Containers 
+
+[abstract]
+{description}
+
+== Do we support modifying the official Couchbase container?
+
+No. The official Couchbase container images on DockerHub and Red Hat Registr are actual binaries which get built with our https://github.com/docker-library/docs/tree/master/couchbase[Dockerfile], which consists of commands to assemble the image and some best practices.
```
Comment (malarky) [inline modules/cloud/pages/couchbase-containers-faq.adoc:9]: Typo on Registry.

**Comment by amarantha-k** [comment]
Thanks! I've incorporated your feedback.

File: modules/cloud/pages/couchbase-containers-faq.adoc [hunk 2]
```diff
@@ -0,0 +1,32 @@
+= Frequently Asked Questions about Couchbase Containers
+:description: Frequently asked questions about Couchbase Containers 
+
+[abstract]
+{description}
+
+== Do we support modifying the official Couchbase container?
```
Comment (b33f) [inline modules/cloud/pages/couchbase-containers-faq.adoc:7]: "Do we support"... reads a bit strange as a customer, who is the we ?  me the customer ?  
IMHO,  "Do Couchbase support"... reads better.

**Pull request merged**